src/main/openapi/common/v1/common-v1.yaml

```diff
@@ -12,39 +12,35 @@ paths:
       description: This resource is only available to supervision users
       operationId: checkHealth
       externalDocs:
-        url: 'https://www.gcloud.belgium.be/rest/#health'
+        url: "https://www.gcloud.belgium.be/rest/#health"
       security: []
       responses:
-        '200':
+        "200":
           description: The service is UP or DEGRADED
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/HealthStatus'
-              examples:
-                response:
```

Comment:
"examples" (allowing multiple examples according to https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject ) could stay so it can be extended with multiple examples.

but I'd rename "response" to "responseUp" to be more descriptive

Reply:
My reasoning was that if we only put a single example, we might as well use the more concise `example:` syntax instead of the named `examples:`. Especially if that also means not having to think about an appropriate descriptive name for the example ;-)

But it's true that adding more examples later is more trivial if you already use the `examples:` by default.

I'm fine with either option. So I'll revert it then?

Reply:
Yeah, not much difference between the two options. I'd revert then with more descriptive name "responseUp"